src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
generally using statics is a bad practice
```suggestion
    private final Random random = new Random();

    public Color getRandomColor() {
```